changedetectionio/static/js/tbltools.js

```diff
@@ -0,0 +1,240 @@
+      y = rows[i + 1].getElementsByTagName("TD")[n];
+      x = x.innerHTML.toLowerCase();
+      y = y.innerHTML.toLowerCase();
+	  /* handle # columns */
+	  if (!isNaN(x)) { 
+		x = parseFloat(x);
+		y = parseFloat(y);
+	  }
+	  /*check if the two rows should switch place,
+      based on the direction, asc or desc:*/
+      if (dir == "asc") {
+		if (x > y) {
+          //if so, mark as a switch and break the loop:
+          shouldSwitch= true;
+          break;
+        }
+      } else if (dir == "desc") {
+        if (x < y) {
+          //if so, mark as a switch and break the loop:
+          shouldSwitch = true;
+          break;
+        }
+      }
+    }
+    if (shouldSwitch) {
+      /*If a switch has been marked, make the switch
+      and mark that a switch has been done:*/
+      rows[i].parentNode.insertBefore(rows[i + 1], rows[i]);
+      switching = true;
+      //Each time a switch is done, increase this count by 1:
+      switchcount ++;      
+    } else {
+      /*If no switching has been done AND the direction is "asc",
+      set the direction to "desc" and run the while loop again.*/
+      if (switchcount == 0 && dir == "asc") {
+        dir = "desc";
+        switching = true;
+      }
+    }
+  }
+
+  // hide all asc/desc sort arrows
+  sortimgs = document.querySelectorAll('[id^="sort-"]');
+  for (var i = 0; i < sortimgs.length; i++) {
+	sortimgs[i].style.display = "none";
+  }
+
+  // show current asc/desc sort arrow
+  if (dir == "asc") {
+    document.getElementById("sort-" + n + "a").style.display = "";
+  }
+  else {
+	document.getElementById("sort-" + n + "d").style.display = "";
+  }
+
+  // show all sortable indicators
+  sortableimgs = document.querySelectorAll('[id^="sortable-"]');
+  for (var i = 0; i < sortableimgs.length; i++) {
+    sortableimgs[i].style.display = "";
+  }
+  
+  // hide sortable indicator from current column
+  document.getElementById("sortable-" + n).style.display = "none";
+
+}
+
+// check/uncheck all checkboxes
+function checkAll(e) {
+	var checkboxes = document.getElementsByName('check');
+	
+	if (e.checked) {
+		for (var i = 0; i < checkboxes.length; i++) {  
+			checkboxes[i].checked = true;
+		}
+	} else {
+		for (var i = 0; i < checkboxes.length; i++) {
+			checkboxes[i].checked = false;
+		}
+	}
+}
+
+// check/uncheck the checkall checkbox if the all checkboxes are checked or unchecked
+function checkChange(){
+	
+	var totalCheckbox = document.querySelectorAll('input[name="check"]').length;
+	var totalChecked = document.querySelectorAll('input[name="check"]:checked').length;
+	
+	// Total checkboxes equals to total checked checkboxes
+	if(totalCheckbox == totalChecked) {
+		document.getElementsByName("showhide")[0].checked=true;
+	} else {
+		document.getElementsByName("showhide")[0].checked=false;
+	}
+}
+
+// search watches in Title column
+function tblSearch(evt, input) {
+
+  var code = evt.charCode || evt.keyCode;
+  if (code == 27) {

```

Comment:
I think there is like EVENT_ENTER_KEY or something instead of `int` right? I would prefer that